src/user/api/serializers.py

```diff
@@ -0,0 +1,24 @@
+from rest_framework import serializers
+from django.contrib.auth import get_user_model,authenticate
+
+User=get_user_model()
+
+class UserSerializer(serializers.ModelSerializer):
```

Comment:
remove this

Reply:
which one??
user=get_user_model() or class userserializer ???

Reply:
userserializer